apps/mobile/src/store/settings/settings.ts

```diff
@@ -54,8 +57,8 @@ export const initialState: SettingsState = {
   accountDisplayPreference: 'native-segwit',
   analyticsPreference: 'consent-given',
   bitcoinUnitPreference: 'bitcoin',
+  emailAddressPreference: '',
   fiatCurrencyPreference: 'USD',
-  createdOn: new Date().toISOString(),
```

Comment:
Why is this removed? This is useful metadata to keep even if we don't display anywhere.

Reply:
I searched and it wasn't being used, so I thought it was just done temporarily for dev purposes. I can add it back. I think at one point it was being used as a placeholder where designers want us to show the app version number? We have the same prop on wallet state.